### DISCUSSION TITLE
ExcludedResourceWarning

### DISCUSSION THREAD

**Discussion by ghost** [argoproj/argo-cd#8674] [Q&A] [answered]
Hello,

I'm working on a project which uses application of applications based on helm charts and ArgoCD. when I Deployed my application of applications I got this warning "ExcludedResourceWarning Resource argoproj.io/Application mysampleapp is excluded in the settings".

Could you please help me with this warning and explain to me what does this warning stand for?? thank you so much

**Comment by todaywasawesome** [comment, marked as answer]
It sounds like something is listed in your exclusions. Checkout your argocd configmap with `kubectl get configmap argocd-cm -n argocd -o yaml` 

https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#resource-exclusioninclusion
